### ISSUE TITLE
(Minor typo) in first Arian's Home scene

### ISSUE THREAD

**Issue by CalCoast** [herp-a-derp/Corruption-of-Champions#834]
"Clearing his throat, Arians yells, _“Boon!”_" 

Seen after visiting Arian's home for the first time. Should be "-Arian yells, _"Boon!"_" I'll add any further typos I find in the next while _(if any)_ to this thread, and if so, I'll change its title.

**Comment by Kitteh6660** [comment]
I could easily fix most of those issues. 1-5, 7-13. What version are you using?

As for number 6, I've looked and have yet to find this bug. It could result from that specific output failing to execute.
Number 14, Lethice scenes are unfortunately not in the main CoC repository. I could fix it in Vanilla+ though.
Number 15, it interestingly chooses either chest or bosom depending on cup size.

I could then make a pull request.

**Comment by CalCoast** [comment]
Right, I forgot that I wasn't playing the original game! Sorry about that. Ctrl+C, ctrl+V: 

"Corruption of Champions (1.0.2_v+_01 (Vanilla Plus!)) Release Build" 

Thanks for your work by the way, Kitteh.
